frontend/src/styles.css

```diff
@@ -4,13 +4,21 @@
  */
 
 /* You can add global styles to this file, and also import other style files */
+
+@import "~@angular/material/prebuilt-themes/indigo-pink.css";
+
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
 
+* {
+  font-family: Roboto, "Helvetica Neue", sans-serif;
+}
+
```

Comment:
This has some unwanted side-effects. I think we need to revert the change. 
The CSS rule applies the font-family on all elements, also the elements where another font is defined.

I just found two cases where this an issue: 
- The tailwind class `font-mono` in DisplayValueComponent doesn't apply anymore, it's overwritten by this rule.
- The font in the monaco editor is broken, go to `Menu` > `Settings` > `Configuration`